tokens/main.py

```diff
@@ -9,7 +9,7 @@
 
 ALLOWED_REPOS = {
     'ancestry': ['ancestry'],
-    'tob-wgs': ['ancestry', 'tob-wgs'],
+    'tob-wgs': ['ancestry', 'tob-wgs', 'joint-calling'],
```

Comment:
Can we keep it sorted lexicographically?

```suggestion
    'tob-wgs': ['ancestry', 'joint-calling', 'tob-wgs'],
```